File: npm/monosecret__client/examples/quick_start.js
const { Monosecret } = require("monosecret");

const resolved = Monosecret.builder()
  .withProvider("keyring://")
  .withProfile("production")
  .withReason("boot web app")
  .load();

console.log(resolved.provider, resolved.profile);
const db = resolved.secrets.DATABASE_URL;
console.log(db.get()); // the value, or the file path for as_path secrets
resolved.setAsEnv(); // export everything into process.env

File: python/monosecret_py/examples/quick_start.py
from monosecret import Monosecret

resolved = (
    Monosecret.builder()
    .with_provider("keyring://")
    .with_profile("production")
    .with_reason("boot web app")
    .load()
)

print(resolved.provider, resolved.profile)
db = resolved.secrets["DATABASE_URL"]
print(db.get)              # the value, or the file path for as_path secrets
resolved.set_as_env()      # export everything into os.environ
